Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -313,6 +350,19 @@ private void RegisterFailure(string instrumentationKey, Exception ex)
             AppMapCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
         }
 
+        /// <summary>
+        /// FetchAppIdFromService failed.
+        /// Registers failure for further action in future.
+        /// </summary>
+        /// <param name="instrumentationKey">Instrumentation key for which the failure occurred.</param>
+        /// <param name="httpStatusCode">Response code from AppId Endpoint.</param>
+        private void RegisterFetchFailure(string instrumentationKey, HttpStatusCode httpStatusCode)
+        {
+            this.failingInstrumenationKeys[instrumentationKey] = new FailedResult(DateTime.UtcNow);
```

Comment:
will it ever be set multiple times? Will it behave OK when you attempt to override value with the new one?

Reply:
Removing this. I verified that the encapsulating code is performing this step.